resources/js/processes-catalogue/components/ProcessInfo.vue

```diff
@@ -22,33 +22,35 @@
               :current-user-id="currentUserId"
               :is-documenter-installed="isDocumenterInstalled"
             />
+            <processes-carousel
+              :process="process"
+              :interval="intervalValue"
```

Comment:
prop process is going to be necessary after, because in component carousel we need processID to build query for API. Interval variable is going to be removed.